Main_project_Helpers.rb

```diff
@@ -27,91 +28,106 @@ def register_user(user_data)
 
 
 	def log_out
-		@driver.find_element(:css, '.logout').click
+		@driver.find_element(:class, 'logout').click
 	end
 
+
 	def log_in(user)
-		@driver.find_element(:css, '.login').click
-		@driver.find_element(:css, '#username').send_keys user.login
-		@driver.find_element(:css, '#password').send_keys user.password
+		@driver.find_element(:class, 'login').click
+		@driver.find_element(:id, 'username').send_keys user.login
+		@driver.find_element(:id, 'password').send_keys user.password
 		@driver.find_element(:css, '[name="login"]').click
 	end
 	
+
  	def change_pass(user)		
- 		@driver.find_element(:css, '.my-account').click
- 		@driver.find_element(:css, '.icon-passwd').click
- 		@driver.find_element(:css, '#password').send_keys user.password
- 		@driver.find_element(:css, '#new_password').send_keys user.change_password
- 		@driver.find_element(:css, '#new_password_confirmation').send_keys user.password
+ 		@driver.find_element(:class, 'my-account').click
+ 		@driver.find_element(:class, 'icon-passwd').click
+ 		@driver.find_element(:id, 'password').send_keys user.password
+ 		@driver.find_element(:id, 'new_password').send_keys user.change_password
+ 		@driver.find_element(:id, 'new_password_confirmation').send_keys user.password
  		@driver.find_element(:css, '[name="commit"]').click
  	end
  	
- 	def create_new_project(project_data)
 
+ 	def create_new_project(project_data={})
  		project = Project.new unless project_data.empty?
 		project = project_data.empty? ? Project.new : Project.new(project_data)
```

Comment:
you should choose which line to delete this or previous one
both of them assign a value to 'project' variable